tests/gui/test_widgets.py

```diff
@@ -1161,12 +1161,14 @@ def test_FileWidget_mousePressEvent_download(mocker):
     """
     mock_controller = mocker.MagicMock()
     mock_signal = mocker.MagicMock()  # not important for this test
-    file_ = factory.File(is_downloaded=False)
-
+    file_ = factory.File()
     session_maker = mocker.MagicMock()
     fw = FileWidget(session_maker, file_, mock_controller, mock_signal)
+    fw.file_is_downloaded = False
```

Comment:
Instead of doing this update here, it would make more sense to just pass `is_downloaded=None, is_decrypted=None` to the factory to ensure those fields are set as we expect. That would be closer to real world behavior.

Reply:
Resolving now that this is outdated after removing session code from gui.